tiff_file_read_benchmark.py

```diff
@@ -0,0 +1,47 @@
+import tifffile as tiff
```

Comment:
As we discussed, yes either remove this file from this PR or discuss with @StephenNneji if could be added elsewhere for future refererence